### PR TITLE
ACMS-1275: Add CI/CD Pipeline to ACMS Article module.

### PR DESCRIPTION
**Pull request by rajeshreeputra**

### CODE REVIEW

File: .github/workflows/acquia_cms_article.yml [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+        if: ${{ success() }}
+        run: ../orca/bin/ci/after_success.sh
+
+      # Display debugging information in case of failure.
+      - name: After failure
+        if: ${{ failure() }}
+        run: ../orca/bin/ci/after_failure.sh
+  VERIFY_ACMS_INSTALL_ON_DRUPAL_PROJECT:
+    if: ${{ github.event_name == 'push' }}
+    name: "Verify commands on acquia/drupal-recommended-project"
```
Comment (chandan-singh7929) [inline .github/workflows/acquia_cms_article.yml]: Change `commands` to `acquia_cms_article`.

File: tests/packages_alter.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+drupal/acquia_cms_article:
+  type: drupal-module
+
+# Do not install Lightning modules explicitly excluded by ACMS.
```
Comment (chandan-singh7929) [inline tests/packages_alter.yml:4]: Please update comment

File: .github/workflows/acquia_cms_article.yml [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+      # Conditionally send code coverage data to Coveralls.
+      - name: After success
+        if: ${{ success() }}
+        run: ../orca/bin/ci/after_success.sh
+
+      # Display debugging information in case of failure.
+      - name: After failure
+        if: ${{ failure() }}
+        run: ../orca/bin/ci/after_failure.sh
+  VERIFY_ACMS_INSTALL_ON_DRUPAL_PROJECT:
```
Comment (chandan-singh7929) [inline .github/workflows/acquia_cms_article.yml:78]: Update job name here.

**Review by chandan-singh7929 — APPROVED** [review]

File: .github/workflows/acquia_cms_article.yml (hunk 3 above)
Comment (rajeshreeputra) [reply .github/workflows/acquia_cms_article.yml]: updated

File: tests/packages_alter.yml (hunk 2 above)
Comment (rajeshreeputra) [reply tests/packages_alter.yml]: updated

**Pull request merged**